src/EMS/RingdownStatus.js

```diff
@@ -19,23 +20,58 @@ function RingdownStatus({ className, onStatusChange, ringdown }) {
     onStatusChange(ringdown, Ringdown.Status.REDIRECTED);
   }
 
+  useEffect(() => {
+    switch (ringdown.payload.patientDelivery.currentDeliveryStatus) {
+      case 'RECEIVED':
+        setRingdownStatus('Delivered');
+        break;
+      case 'ARRIVED':
+        setRingdownStatus('Confirmed');
+        break;
+      case 'OFFLOADED':
+        setRingdownStatus('Pending');
+        break;
+      default:
+        break;
+    }
+  }, [ringdown.payload.patientDelivery.currentDeliveryStatus]);
+
```

Comment:
Creating a new state variable and a useEffect callback simply for mapping the status to a string seems like overkill. I'm going to remove it and replace it with some simple conditionals in the output markup.